ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -615,6 +636,14 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
         "com.github.ambry.store.CompactAllPolicyFactory");
     storeMinLogSegmentCountToReclaimToTriggerCompaction =
         verifiableProperties.getIntInRange("store.min.log.segment.count.to.reclaim.to.trigger.compaction", 1, 1, 1000);
+    storeStatsBasedMiddleRangeCompactionIntervalInMs =
```

Comment:
Can we have a reasonable default value here? How often are we planning to run it currently?

Reply:
I use this as a "enable" flag so I don't need another "enable" flog.
I probably will start with 2 day as our full compaction frequency. Ideally we will get rid of this "time based" frequency after we tune it well.